### PR TITLE
User interface updates.

### PR DESCRIPTION
**Pull request by xcguy**
Fix issue with acceleration tile.
Degree symbol on temperature.
Use fixed lux unit (so it displays correctly in logs, activity
messages).
Round temperature for display.
Minor cleanups in help text.

### CODE REVIEW

File: device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy [hunk 1]
```diff
@@ -100,14 +100,14 @@ metadata {
...
         }
         valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat") {
             state "battery", label:'${currentValue}% battery', unit:""
         }
         standardTile("acceleration", "device.acceleration") {
-            state("vibration", label:'${currentValue}', icon:"st.motion.acceleration.active", backgroundColor:"#53a7c0")
-            state("still", label:'${currentValue}', icon:"st.motion.acceleration.inactive", backgroundColor:"#ffffff")
+            state("active", label:'Vibration', icon:"st.motion.acceleration.active", backgroundColor:"#53a7c0")
+            state("inactive", label:'Still', icon:"st.motion.acceleration.inactive", backgroundColor:"#ffffff")
         }
```
Comment (CyrilPeponnet) [inline device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy:114]: Actually this tile is working when you set your device to vibration mode.

**Comment by CyrilPeponnet** [comment]
Looks good except for the vibration tile.

File: device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy (hunk 1 above)
Comment (xcguy) [inline device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy:114]: Initially, my unit reported 'inactive'.   After your comment, I knocked on the sensor to trigger the event and it did indeed show numbers (acceleration), and then finally 'still'.  
I'd suggest keeping your code, but adding the one state for 'inactive' to cover that one end case.

Comment (CyrilPeponnet) [inline device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy:114]: Make sense. Go ahead.

**Comment by xcguy** [comment]
Revised the pull, should cover all the possible states.

Nice device code, much better than the earlier attempts by others!

**Comment by CyrilPeponnet** [comment]
Thanks I appreciate :)

**Pull request merged**